front/src/common/redux/groups/groups.selectors.js

```diff
@@ -0,0 +1,6 @@
+export const updateGroupInGroupList = (groupList, newGroupData) =>{
```

Comment:
It is not a selector, the selector should only convert data from new format to another or get some parts of it, it should not modify it

Reply:
Data should be changed only inside reducer via actions

Reply:
done, selectors file is deleted